internal/provider/provider.go

```diff
@@ -81,6 +80,9 @@ func (p *provider) Configure(ctx context.Context, req tfsdk.ConfigureProviderReq
 
 	config := dvc_mgmt.NewConfiguration()
 	config.AddDefaultHeader("Authorization", p.AccessToken)
+	config.AddDefaultHeader("dvc-referrer", "terraform")
+	metadata := fmt.Sprintf("{\"dvc_terraform_provider_version\": \"%s\", \"terraform_version\": \"%s\"}", p.version, req.TerraformVersion)
```

Comment:
@JamieSinn PR looks good to me, can you verify these two versions are being pulled the right way? :)

Reply:
yep - that's correct - but we need to ensure that the acceptance tests are passing correctly. 
We may need to re-build the test org?

Reply:
Updated the `BasePath` so it actually hits the Bucketing API correctly... how do things look now? 👀